projects/addon-doc/src/components/tab-content/tab-content.template.less

```diff
@@ -0,0 +1,34 @@
+@import 'taiga-ui-local';
+
+@keyframes fadeIn {
+    0% {
+        opacity: 0;
+    }
+
+    100% {
+        opacity: 1;
+    }
+}
+
+.t-use-fade-animate {
+    will-change: opacity;
+    animation: fadeIn ease-in 500ms;
+    animation-fill-mode: forwards;
+    animation-iteration-count: 1;
+}
+
+.t-demo {
+    .customize-scroll();
+    padding: 2rem;
+    overflow-x: auto;
+
+    @media @mobile-m {
```

Comment:
can we use host-context for detect mobile like in this [PR](https://github.com/Tinkoff/taiga-ui/pull/1130/files)?